src/test/java/uk/gov/hmcts/reform/coh/util/QuestionStateUtils.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.coh.util;
+
+import uk.gov.hmcts.reform.coh.domain.QuestionState;
+import uk.gov.hmcts.reform.coh.states.QuestionStates;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
```

Comment:
This will create instances of QuestionState, so the test classes don't need to keep creating new instances of QuestionState. It looks round QuestionStates enums and creates a dummy instance of QuestionState.